redis/src/test/scala/zio/redis/StreamsSpec.scala

```diff
@@ -1230,6 +1230,44 @@ trait StreamsSpec extends BaseSpec {
             result <- xTrim(stream, 1000L).either
           } yield assert(result)(isLeft(isSubtype[WrongType](anything)))
         }
+      ),
+      suite("XInfo")(
+        testM("stream info") {
```

Comment:
I suggest that you add test cases for a non-existent stream and for a key that doesn't reference a stream.